templates/connect.php

```diff
@@ -325,9 +357,18 @@ class="fs-permission fs-<?php esc_attr_e( $id ); ?>">
 		 * @author Vova Feldman (@svovaf)
 		 * @since 1.1.9
 		 */
-		var $connectLicenseModeTrigger = $('#fs_connect .fs-freemium-licensing a');
+		var
+			$connectLicenseModeTrigger = $('#fs_connect .fs-freemium-licensing a'),
+			href = window.location.href;
+
+		if ( -1 === href.indexOf( '?' ) ) {

```

Comment:
@fajardoleo I think that `indexOf` return different values in different browsers. I would instead use `if ( 0 < href.indexOf( '?' ) )` and switch the logic between the if and else.